providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/Config.java

```diff
@@ -11,7 +11,13 @@
 @Slf4j
 public final class Config {
     static final Resolver DEFAULT_RESOLVER_TYPE = Resolver.RPC;
-    static final String DEFAULT_PORT = "8013";
+    static final String DEFAULT_RPC_PORT = "8013";
+    /**
+     * @deprecated Use {@link Config#DEFAULT_RPC_PORT} or {@link Config#DEFAULT_IN_PROCESS_PORT}
+     */
+    @Deprecated
+    static final String DEFAULT_PORT = DEFAULT_RPC_PORT;
```

Comment:
nit - IMO this can be removed as we use this internally besides its package private :)